.maestro/subflows/repay.yaml

```
@@ -0,0 +1,27 @@
+appId: ${APP_ID ?? "app.exactly"}
+---
+- scrollUntilVisible: { element: Repay }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)